hack/openshift/sync.sh

```diff
@@ -1,3 +1,28 @@
+#!/bin/bash
```

Comment:
nit: would you mind to update the commit title to address what is being included in it? thanks

Reply:
done.